.github/workflows/dockerhub-publish.yml

```diff
@@ -1,9 +1,6 @@
-name: Publish Image to Docker Hub
+name: Build Docker Image and Push to Docker Hub on Tags
 
-on:
```

Comment:
the previous config limited this workflow to run only on tagged with `v*`

this change make it could be triggered by all pushes and will push to docker registry only on the main repo with `v*` tags

Reply:
it would build the docker image on every push, but ONLY push to docker registry with tag `v*`.

it's to make sure we could build the container without problems.

but it seems necessary to limit the build frequency due to the long build time, maybe I could limit it to the files related to docker

Reply:
I am tired of fighting with CI syntax 😅, changed this back to only trigger on `v*` tags,

but also add a condition to make sure only run this on `CeresDB/` org.

let's keep the docker build validation on a new PR https://github.com/CeresDB/ceresdb/pull/150